starter-code/scripts/article.js

```diff
@@ -3,8 +3,15 @@
 var articles = [];
 
 function Article (rawDataObj) {
```

Comment:
The name of the object in this parameter needs to match the name inside the function or it won't work. I think you should whiteboard how plugging an object into a function would work in referencing the object properties.